src/host/ft_uia/Elements/CmdApp.cs

```diff
@@ -410,10 +410,11 @@ private void ExitCmdProcess()
 
             this.UIRoot = null;
 
-            if (this.job != IntPtr.Zero)
+            if (this.job == IntPtr.Zero)
             {
-                WinBase.TerminateJobObject(this.job, 0);
+                return;
             }
+            WinBase.TerminateJobObject(this.job, 0);
             this.job = IntPtr.Zero;

```

Comment:
Same here. I feel like this is easier to read:
```suggestion
            if (this.job != IntPtr.Zero)
            {
                WinBase.TerminateJobObject(this.job, 0);
                this.job = IntPtr.Zero;
            }
```